keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -313,6 +320,52 @@ class TitusClusterHandler(
       false
     }
 
```

Comment:
This is the heuristic part, where we handle if the only diff is that there are too many enabled server groups